backend/pick-git/src/test/java/com/woowacourse/pickgit/config/TestInfrastructureConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package com.woowacourse.pickgit.config;
+
+import com.woowacourse.pickgit.common.mockapi.MockPickGitStorage;
+import com.woowacourse.pickgit.common.mockapi.MockRepositoryApiRequester;
+import com.woowacourse.pickgit.common.mockapi.MockTagApiRequester;
+import com.woowacourse.pickgit.post.domain.PickGitStorage;
+import com.woowacourse.pickgit.post.infrastructure.PlatformRepositoryApiRequester;
+import com.woowacourse.pickgit.tag.infrastructure.PlatformTagApiRequester;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+
+@TestConfiguration
+public class TestInfrastructureConfiguration {
```

Comment:
```suggestion
public class InfrastructureTestConfiguration {
```

클래스에 `@TestConfiguration` 어노테이션이 붙으니까 네이밍을 Test <-> Infrastructure 위치 바꿔서 하는 건 어떤가요??